deliver/lib/deliver/runner.rb

```diff
@@ -266,7 +266,7 @@ def submit_for_review
     # If itc_provider was explicitly specified, use it.
     # If there are multiple teams, infer the provider from the selected team name.
     # If there are fewer than two teams, don't infer the provider.
-    def transporter_for_selected_team(upload: false)
+    def transporter_for_selected_team(altool_compatible_command: false)
```

Comment:
What's the situation `altool_compatible_command` to be `false`?

I think this flag is not necessary for `ItunesTransporter`.

In the previous implementation, `verify` task doesn't use `altool`, so some flags are needed to decide which executors are used.
However, in the current implementation, the transporter always uses `altool` when meeting the situations.
So I think this is not needed. What do you think?

Reply:
I meant all callers of this method seem to pass `true` for `altool_compatible_command`

https://github.com/fastlane/fastlane/pull/20738/files#diff-51e845b40462cd52020ab428469c85f65e24e1343e0b17c88d0c1a11e1ec22ecR172
https://github.com/fastlane/fastlane/pull/20738/files#diff-51e845b40462cd52020ab428469c85f65e24e1343e0b17c88d0c1a11e1ec22ecR209

So we can get rid of this argument. We can always pass `true` on L284 to reduce unnecessary states.